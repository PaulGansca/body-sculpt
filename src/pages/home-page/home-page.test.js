import { shallow } from 'enzyme';
import React from 'react'
import HomePage from './home-page';

let wrapper;

beforeEach(() => {
    wrapper = shallow(<HomePage />);
})

describe('<HomePage />', () => {
  it("expect to render home page", () => {
    expect(wrapper).toMatchSnapshot();
  });
});