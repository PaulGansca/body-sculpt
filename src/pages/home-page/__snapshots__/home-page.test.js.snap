// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> expect to render home page 1`] = `ShallowWrapper {}`;
